pkg/scheduler/framework/plugins/dynamicresources/dynamicresources_test.go

```diff
@@ -1497,6 +1502,67 @@ func Test_isSchedulableAfterClaimChange(t *testing.T) {
 	}
 }
 
+func Test_isSchedulableAfterPodChange(t *testing.T) {
+	testcases := map[string]struct {
+		objs         []apiruntime.Object
+		pod          *v1.Pod
+		claims       []*resourceapi.ResourceClaim
+		obj          interface{}
+		expectedHint framework.QueueingHint
+		expectedErr  bool
+	}{
+		"backoff-wrong-new-object": {
+			pod:         podWithClaimTemplate,
+			obj:         "not-a-claim",
+			expectedErr: true,
+		},
+		"complete": {
+			objs:         []apiruntime.Object{pendingClaim},
```

Comment:
we don't check these objects in the function, do we?

Reply:
We do, indirectly in https://github.com/kubernetes/kubernetes/blob/3d6c5b2e98afaaae1d17107e2d3d709c726be49d/pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go#L683-L686

As part of my work on https://github.com/kubernetes/kubernetes/pull/127277 I checked whether removing it would make a difference, but couldn't measure any, so I decided to keep the `Get` as an additional check.